README.md

```diff
@@ -3,7 +3,72 @@
 This repo is dedicated to mirror images from other organizations into Rancher.
 There are no packaging changes or changes in the layers of these images.
 
-## Mirroring images
+> [!IMPORTANT]
+> This repository is currently undergoing an overhaul in order to accomplish the
+> goals laid out in
+> [EDR 005](https://docs.google.com/document/d/1IVu4Lq13yKiSs0gl_tIJ3vuidVmt0FIkO2mlZXTRAH4/edit?pli=1&tab=t.0#heading=h.k15gkmaregsk).
```

Comment:
Should we have a private document linked here?

Reply:
@diogoasouza I asked @prachidamle to weigh in here.

@rohitsakala the EDR is indeed not marked approved. But this is not because we haven't done the work of vetting the proposal. I've gone through the process of holding a meeting, responding to comments, et cetera; all stakeholders have given their input and changes have been made accordingly. I then reached out to a variety of people in order to try to get the EDR marked approved before opening PRs, but that did not go anywhere. Because of that, and because these changes do not break anyone else's projects, I'm moving forward. If there are any problems then I will no doubt hear about them.

Reply:
Yes we should not merge untill EDR is approved - this should treated more as a POC until then.

Reply:
About private docs - lets not link any internal docs please, just like with any JIRA issues. @adamkpickering